db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogiteminitialization.rb

```diff
@@ -40,13 +40,31 @@ def create_category_and_tags_if_necessary(dialog_tags_hash)
 def override_service_name(dialog_options_hash)
   log_and_update_message(:info, "Processing override_service_name...", true)
   new_service_name = dialog_options_hash.fetch(:service_name, nil)
-  new_service_name = "#{@service.name}-#{Time.now.strftime('%Y%m%d-%H%M%S')}" if new_service_name.blank?
+  new_service_name = "#{@service.name}-#{Time.zone.now.strftime('%Y%m%d-%H%M%S')}" if new_service_name.blank?
 
   log_and_update_message(:info, "Service name: #{new_service_name}")
   @service.name = new_service_name
   log_and_update_message(:info, "Processing override_service_name...Complete", true)
 end
 
+def override_service_retirement_days(dialog_options_hash)
```

Comment:
@billfitzgerald0120 @tinaafitz 
Should this be moved into the app/models or the retirement mixin?